### ISSUE TITLE
error: unrecognized arguments: --config-file .\hoyo.toml

### ISSUE THREAD

**Issue by sonicjhon1** [atomicptr/hoyo-daily-logins-helper#7]
Couldn't use config-file as shown in the README.
```
$ hoyo-daily-logins-helper --config-file .\hoyo.toml
usage: hoyo-daily-logins-helper [-h] [-c COOKIE] [--genshin] [--starrail] [--debug]
hoyo-daily-logins-helper: error: unrecognized arguments: --config-file .\hoyo.toml
```

The **-h** flag also doesn't document about the **--config-file** flag
```
$ hoyo-daily-logins-helper -h
usage: hoyo-daily-logins-helper [-h] [-c COOKIE] [--genshin] [--starrail] [--debug]
```

**Comment by atomicptr** [comment]
Your version is probably outdated 🤔

**Comment by atomicptr** [comment]
I presume that this is/was the case, if not feel free to re-open the ticket :)!

**Comment by sonicjhon1** [comment]
Yep, fixed by upgrading to python 3.11 and reinstalling the helper, thank you ♥️
